### DISCUSSION TITLE
Tramp support

### DISCUSSION THREAD

**Discussion by ptonner** [wyuenho/emacs-pet#18] [Ideas]
I started playing around with basic support for tramp here: https://github.com/ptonner/emacs-pet/commit/0fefc83ecb7d3b22dfd51fe0a9058305412f76e6. With this patch, calls like `(pet-executable-find "ipython")` work as intended on remote hosts (at least under minimal proof of concept testing).

I'm guessing there are other places to be updated too for full tramp support. I can try to include those as well if given some pointers to their spot in the code

**Comment by wyuenho** [comment]
Tramp support should be somewhat decent now.
